yarn-project/archiver/src/archiver/archiver_store.ts

```diff
@@ -31,18 +32,12 @@ export interface ArchiverDataStore {
   getL2Blocks(from: number, take: number): Promise<L2Block[]>;
 
   /**
-   * Append new encrypted logs to the store's list.
-   * @param data - The encrypted logs to be added to the store.
+   * Append new logs to the store's list.
+   * @param data - The logs to be added to the store.
+   * @param logType - The type of the logs to be added to the store.
    * @returns True if the operation is successful.
    */
-  addEncryptedLogs(data: L2BlockL2Logs[]): Promise<boolean>;
-
-  /**
-   * Append new unencrypted logs to the store's list.
-   * @param data - The unencrypted logs to be added to the store.
-   * @returns True if the operation is successful.
-   */
-  addUnencryptedLogs(data: L2BlockL2Logs[]): Promise<boolean>;
+  addLogs(data: L2BlockL2Logs[], logType: 'encrypted' | 'unencrypted'): Promise<boolean>;
```

Comment:
Shouldn't this be an enum now?

Reply:
You are right. Addressed in [4a6388a](https://github.com/AztecProtocol/aztec-packages/pull/929/commits/4a6388aabcdd31cefc9ee18d476126ea2f8d82f6)